packages/zarith_stubs_js/zarith_stubs_js.v0.15.0/opam

```diff
@@ -14,7 +14,7 @@ depends: [
   "dune"  {>= "2.0.0"}
 ]
 # The conflict is a run-time failure when versions do not match:
-conflicts: "zarith" {< "1.12"}
+conflicts: [ "zarith" {< "1.12" | >= "1.13" } ]
```

Comment:
```suggestion
conflicts: [ "zarith" {< "1.12" | >= "1.13" } ]
```
Is there any compatible zarith version in this case?

Reply:
I guess zarith.1.12 is the only version compatible with that release of zarith-stubs-js. But I'm a bit tired so I don't really trust myself on having gotten the comparison and operators right.

Btw, the previous constraint is from https://github.com/ocaml/opam-repository/pull/21056

Reply:
I was sleeping, I saw equal signs on both sides :)

Reply:
```suggestion
conflicts: [ "zarith" {!= "1.12" } ]
```

Reply:
I pushed essentially this but added a comment too

Reply:
The original < and >= was a better constraint, since it allows for any variants of zarith.1.12 to also match (e.g. `zarith.1.12+dune`, that sort of thing)

Reply:
Reverted to `<` and `>=`